### PR TITLE
inlg 2022 ingestion

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Comment by ThiagoCF05** [comment]
Dear all,

Any news about the merge?

**Comment by mjpost** [comment]
I've been waiting on #1896 to be completed since it introduces a large number of substantive changes. I can merge this in, however, and just handle it manually.

File: data/xml/2022.inlg.xml [hunk 1]
```diff
@@ -0,0 +1,373 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2022.inlg">
+  <volume id="inlg" ingest-date="2022-11-08">
```
Comment (mjpost) [inline data/xml/2022.inlg.xml:3]: Hi @xinru1414—the volume name here should be "main", not "inlg".

File: data/xml/2022.nlg4health.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2022.nlg4health">
+  <volume id="1" ingest-date="2022-11-08">
+    <meta>
```
Comment (mjpost) [inline data/xml/2022.nlg4health.xml:4]: This volume should also be linked under "ws" in `joint.yaml`.

**Review by mjpost — COMMENTED** [review]
Hi @xinru1414, please see comments. When those are addressed, I can merge.

File: data/xml/2022.inlg.xml (hunk 1 above)
Comment (xinru1414) [reply data/xml/2022.inlg.xml:3]: yep i missed that since they are using aclpub2 format

**Comment by ThiagoCF05** [comment]
Dear all,

Is there an estimation when the repo will be merged?

Thanks in advance,
Thiago.

**Comment by mjpost** [comment]
I think I can do this shortly.

**Comment by ThiagoCF05** [comment]
The checks seem to have failed. Is there something wrong?

**Comment by mjpost** [comment]
Sometimes it dies without explanation. I triggered a rebuild.

**Review by mjpost — APPROVED** [review]

**Comment by mjpost** [comment]
Merged. Should be live within an hour unless the build goes awry.

**Pull request merged**